docker/localConfig.example.yml

```diff
@@ -0,0 +1,34 @@
+
+configs:
+  localConfig.cjs:
+    content: |
+      const baseUrl = process.env.BASE_URL || 'http://implementation:8000';
```

Comment:
We don't actually have any known scenarios where having this be a JS file is actually useful.

What do you think about converting it to JSON, so that it can sit natively within the YAML (since JSON is a subset of YAML...they say...)?

Reply:
The issue is that this will require to change the actual test suites behavior. The way I've implemented it here is just plug and play.

We can discuss